packages/ceramic-cli/src/index.ts

```diff
@@ -18,14 +25,39 @@ program
         if (stateStorePath == null) {
             stateStorePath = DEFAULT_PINNING_STORE_PATH
         }
-        await CeramicDaemon.create({
-            ipfsHost: ipfsApi,
+
+        const config = {
             ethereumRpcUrl: ethereumRpc,
             anchorServiceUrl: anchorServiceApi,
             stateStorePath: stateStorePath,
             validateDocs,
             pinning: pinning
-        })
+        }
+
+        if (ipfsApi) {
+            Object.assign(config, {
+                ipfsHost: ipfsApi
+            })
+        } else {
+            multiformats.multicodec.add(dagJose)
+            const format = legacy(multiformats, dagJose.name)
+
+            const ipfs = await Ipfs.create({
```

Comment:
Seems this could simply be set as `config.ipfs = await Ipfs.create({`

Reply:
👍